src/webcomponents/helperfunctions/jsonToCol.ts

```diff
@@ -0,0 +1,124 @@
+export const jsonToCol = (languages: string): any => {
+  let datas: any = JSON.parse(languages);
+  let dataArray: any = [];
+  let keys: any = [];
+  let columns: any = [];
+  let columnsValues: any = [];
+  let rows: any = [];
+  let fullRowData: any = [];
+  let columnTitle: any = [{ type: "text", title: "messageID", width: 150 }];
+
+  const setTitle = () => {
+    if (datas) {
```

Comment:
you need to pass data as parameter to the top level function
minimize use of side effects